teslajsonpy/car.py

```diff
@@ -35,6 +35,18 @@
 }
 
 
+def safeget(dct, *keys, default=None):
+    """Get a recursuive object from a dict."""
+    for key in keys:
+        try:
```

Comment:
Why not use dict.get?

Reply:
I don't use get, otherwise we end up with `.get(...).get(...).get(....)` and it can get quite long and messy.

This little helper function cleans that up.
If you're happy with it, in a later pull request, i can update all the others to use the helper function.